policyengine_us/parameters/gov/irs/credits/refundable.yaml

```diff
@@ -6,15 +6,15 @@ values:
     - refundable_ctc
     - recovery_rebate_credit
     - refundable_payroll_tax_credit
-    - premium_tax_credit
+    #- premium_tax_credit
```

Comment:
Just delete rather than comment

Reply:
You can leave a comment indicating that, and file an issue to add it back in the future. But commented _code_, especially by itself, makes it unclear what the appropriate logic should be.

See this blog post for more: https://kentcdodds.com/blog/please-dont-commit-commented-out-code